apps/common/util/common.py

```diff
@@ -211,3 +211,7 @@ def split_and_transcribe(file_path, model, max_segment_length_ms=59000, audio_fo
         if isinstance(text, str):
             full_text.append(text)
     return ' '.join(full_text)
+
+
+def _remove_empty_lines(text):
+    return '\n'.join(line for line in text.split('\n') if line.strip())
```

Comment:
There are two small issues and one optimization suggestion in the provided code:

### Issues:
1. **Missing `return` Statement**: The function `_remove_empty_lines` is missing a return statement at the end. This should be added to properly return the cleaned string.
   
2. **Potential Memory Leak**: The function currently uses list comprehensions to create new lists (`full_text`) and `' '.join(...)`. While this approach avoids modifying strings directly, it might not lead to efficient memory usage with very large datasets because each operation creates new objects.

### Optimization Suggestion:
Instead of using list comprehensions for operations like filtering or joining strings, consider using generators when possible. Generators can reduce memory consumption since they yield results one at a time rather than creating intermediate lists.
```python
def _remove_empty_lines(text):
    # Use generator expression to filter out empty lines
    non_empty_lines = (line for line in text.split('\n') if line.strip())
    return ''.join(non_empty_lines)
```

These changes will help ensure that the function behaves correctly and efficiently.